ModelCatalogueCorePlugin/grails-app/assets/javascripts/modelcatalogue/core/ui/utils/orderedMapEditor.coffee

```diff
@@ -3,6 +3,7 @@ angular.module('mc.core.ui.utils').directive 'orderedMapEditor',  [-> {
     replace: true
     scope:
       object:             '='
+      canDeprecate:       '='
```

Comment:
@kuceram this should be passed as attribute `@` not to create another unnecessary angular watch